android/src/main/java/com/revenuecat/purchases/common/mappers/mappersHelpers.kt

```diff
@@ -42,6 +42,6 @@ fun JSONObject.convertToMap(): Map<String, String?> =
         }
     }
 
-internal fun Date.toMillis(): Double = this.time.div(1000.0)
+public fun Date.toMillis(): Long = this.time
```

Comment:
I think this bug might have been around forever

Reply:
the iOS equivalent was fixed in 3.0.0, but since this function was called `toMillis` I assumed it was correct 🤦